app/Providers/AppServiceProvider.php

```diff
@@ -62,6 +65,10 @@ public function boot()
         Cookies::essentials()
             ->session()
             ->csrf();
+
+        DB::listen(function (QueryExecuted $query) {
+            Log::debug($query->sql, $query->bindings);
+        });
```

Comment:
Is this intentional in the final changeset?

Reply:
noooo 😄